server/events/models/models.go

```diff
@@ -302,6 +302,8 @@ type ProjectCommandContext struct {
 	// project is in.
 	AutomergeEnabled bool
 	// AutoplanEnabled is true if autoplanning is enabled for this project.
+	ParallelPlansEnabled bool
+	// ParallelPlansEnabled is true if parallel plans is enabled for this project.
```

Comment:
Pretty sure this comment is out of order

Reply:
Yes! Good catch